examples/nginx-with-rspamd/docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+version: "3.5"
+
+services:
+  db:
+    image: mariadb:10.5
+    container_name: anonaddy_db
+    command:
+      - "mysqld"
+      - "--character-set-server=utf8mb4"
+      - "--collation-server=utf8mb4_unicode_ci"
+    volumes:
+      - "./db:/var/lib/mysql"
+    environment:
+      - "MYSQL_ALLOW_EMPTY_PASSWORD=yes"
+      - "MYSQL_DATABASE"
+      - "MYSQL_USER"
+      - "MYSQL_PASSWORD"
+    restart: always
+
+  redis:
+    image: redis:4.0-alpine
+    container_name: anonaddy_redis
+    restart: always
+
+  anonaddy:
+    image: anonaddy/anonaddy:latest
+    container_name: anonaddy
+    depends_on:
+      - db
+      - redis
+    ports:
+      - target: 25
+        published: 25
+        protocol: tcp
+      - target: 8000
+        published: 8000
+        protocol: tcp
+    volumes:
+      - "./data:/data"
+    env_file:
+      - "./anonaddy.env"
+    environment:
+      - "DB_HOST=db"
+      - "DB_DATABASE=${MYSQL_DATABASE}"
+      - "DB_USERNAME=${MYSQL_USER}"
+      - "DB_PASSWORD=${MYSQL_PASSWORD}"
+      - "REDIS_HOST=redis"
+    restart: always
+
+  nginx:
+    image: nginx:1.20.1-alpine
+    container_name: anonaddy_nginx
+    restart: unless-stopped
+    ports:
+      - '443:443'
+    volumes:
+      - ./nginx/dhparam.pem:/etc/ssl/dhparam.pem
+      - ./nginx/templates:/etc/nginx/templates
+      - /etc/letsencrypt:/etc/letsencrypt
+    environment:
+      - NGINX_HOST=example.com
+      - NGINX_PROXY=anonaddy:8000
+    depends_on:
+      - anonaddy
+      - rspamd
+
+  rspamd:
+   image: mailprotector/rspamd:0.1
```

Comment:
I have some concerns about this image. Does not seem to have a lot of traction/contributors: https://github.com/mailprotector/docker-rspamd

Looking at the source code https://github.com/mailprotector/docker-rspamd/blob/main/Dockerfile I think we can install the required pkg `rspamd rspamd-controller rspamd-proxy` ourselves and add a new s6 service for rspamd. This way we would not need an example folder and we could merge these changes in repo root. We could also remove `RSPAMD_HOST` and `RSPAMD_PORT` vars and only have one named `RSPAMD_ENABLE`.

Reply:
this container is has stayed up to date with latest rspamd releases (on alpine) and is pretty simple. with that said, given that opendkim/opendmarc are already coordinated within the container, i makes sense to continue the pattern by installing rspamd internally as well. my preference is for sidecars, but it's a tradeoff. i'll give it a shot.

Reply:
yeah I like sidecars too and use them quite a lot but I think in this case we can simply make rspamd listen to 127.0.0.1 and avoid network overhead on the overlay.